Dockerfile

```
@@ -0,0 +1,29 @@
+FROM ubuntu:xenial
+
+RUN apt update && \
+  apt install -qy git-core language-pack-en libmysqlclient-dev libssl-dev python3.5 python3-pip python3.5-dev && \
+  pip3 install --upgrade pip setuptools && \
+  rm -rf /var/lib/apt/lists/*
```

`apt` warns that its interface isn't stable. Probably better to use `apt-get` here.
```suggestion
RUN apt-get update && \
  apt-get install -qy git-core language-pack-en libmysqlclient-dev libssl-dev python3.5 python3-pip python3.5-dev && \
  pip3 install --upgrade pip setuptools && \
  rm -rf /var/lib/apt/lists/*
```